packages/kbn-dev-utils/src/kbn_client/kbn_client.ts

```diff
@@ -54,8 +54,8 @@ export class KbnClient {
   /**
    * Make a direct request to the Kibana server
    */
-  async request(options: ReqOptions) {
-    return await this.requester.request(options);
+  async request<T>(options: ReqOptions) {
```

Comment:
We need someone from Kibana (??) to look over this change and give us a 👍  - just to make sure they are aware 😬 .